wtfis/models/abuseipdb.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from typing import Optional, List
+
+from pydantic import BaseModel, Field, RootModel
+
+
+class AbuseIpDb(BaseModel):
+    ip_address: str = Field(alias="ipAddress")
+    is_public: Optional[bool] = Field(None, alias="isPublic")
+    ip_version: Optional[int] = Field(None, alias="numDistinctUsers")
+    is_whitelisted: Optional[bool] = Field(None, alias="ipVersion")
+    abuse_confidence_score: int = Field(alias="abuseConfidenceScore")
+    country_code: Optional[str] = Field(None, alias="countryCode")
+    country_name: Optional[str] = Field(None, alias="countryName")
+    usage_type: Optional[str] = Field(None, alias="usageType")
+    isp: str
+    domain: Optional[str] = None
+    hostnames: Optional[List[str]] = None
+    is_tor: Optional[bool] = Field(None, alias="isTor")
+    total_reports: Optional[int] = Field(None, alias="totalReports")
+    num_distinct_users: Optional[int] = Field(None, alias="numDistinctUsers")
+    last_reported_at: Optional[str] = Field(None, alias="lastReportedAt")
+    reports: Optional[List] = None
+
+
+class AbuseIpDbMap(RootModel):
+    root: dict[str, AbuseIpDb]
```

Comment:
Sorry, I missed this one. This should be `Dict[str, AbuseIpDb]`. I'm surprised the tests are passing except in Python 3.8!

Reply:
It made me think if I used vermin, would it show the problem. Sure.

Reply:
It was a 3.9 thing: https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections

Reply:
Solved. I also checked against vermin:

```powershell
vermin --target=3.8 --violations --backport argparse --backport typing --backport ipaddress --backport typing_extensions  --eval-annotations --no-parse-comments .\
```

Reply:
Good catch! Also just learned about vermin - pretty cool!